plugins/org.jkiss.dbeaver.ext.altibase/src/org/jkiss/dbeaver/ext/altibase/AltibaseConstants.java

```diff
@@ -0,0 +1,118 @@
+
+    public static final int PACKAGE_TYPE_SPEC = 6;
+    public static final int PACKAGE_TYPE_BODY = 7;
+    
+    public static final int TYPE_BYTE = 20001;
+    public static final int TYPE_VARBYTE = 20003;
+    public static final int TYPE_NIBBLE = 20002;
+    
+    public static final int TYPE_GEOMETRY = 10003;
+    public static final String TYPE_NAME_GEOMETRY = "GEOMETRY";
+    public static final String TYPE_NAME_TIMESTAMP = "TIMESTAMP";
+    public static final String TYPE_NAME_DATE = "DATE";
+    
+    public static final String OBJ_TYPE_MATERIALIZED_VIEW = "MATERIALIZED VIEW";
+    public static final String OBJ_TYPE_TYPESET = "TYPESET";
+    
+    public static final String NO_DBMS_METADATA = "-- [WARNING] Without DBMS_METADATA package, " 
+            + "the generated DDL may not be correct." + NEW_LINE;
+    
+    public static final String NO_DDL_WITHOUT_DBMS_METADATA = 
+            "-- [WARNING] DBMS_METADATA package is required to have DDL for %s." + NEW_LINE;
+    
+    /*
+     * Preference page
+     */
+    public static final String PREF_EXPLAIN_PLAN_TYPE = "altibase.explain.plan.type";
+    public static final String PREF_DBMS_OUTPUT = "altibase.dbms.output";
+    public static final String PREF_PLAN_PREFIX = "altibase.plan.prefix";
+    
+    public enum ExplainPlan {
+        ONLY("EXPLAIN PLAN = ONLY", (byte) 2),
+        ON("EXPLAIN PLAN = ON", (byte) 1);
+        
+        private String title;
+        private byte value;
+        
+        ExplainPlan(String title, byte value) {
+            this.title = title;
+            this.value = value;
+        }
+        
+        public String getTitle() {
+            return this.title;
+        }
+        
+        public byte getArgValue() {
+            return this.value;
+        }
+        
+        /**
+         * Get EXPLAIN_PLAN enum object by index
+         */
+        public static ExplainPlan getByIndex(int idx) throws ArrayIndexOutOfBoundsException {
+            for (ExplainPlan expPlan : ExplainPlan.values()) {
+                if (expPlan.ordinal() == idx) {
+                    return expPlan;
+                }
+            }
+            
+            throw new ArrayIndexOutOfBoundsException("No such index value in EXPLAIN_PLAN: " + idx);
+        }
+    }
+    
+    public static final String SQL_STATE_TOO_LONG = "22026";
+    
+    /*
+     * Password grace time
+     */
+    public static final String SQL_WARNING_TITILE = "SQL Warning";
+    public static final String PASSWORD_WILL_EXPIRE_WARN_DESCRIPTION = "Change the password or contact the DBA.";
+    
+    public static final int EC_PASSWORD_WILL_EXPIRE = 0x51A79;
+    public static final int EC_DBMS_METADATA_NOT_FOUND = 0xF1B3D;
+
+    /*
+     * DBMS output
+     */
+    // e.g. Altibase.jdbc.driver.AltibaseMessageCallback, Altibase7_1.jdbc.driver.AltibaseMessageCallback
+    public static final String CLASS_NAME_4_CONNECTION_POSTFIX = ".jdbc.driver.AltibaseConnection";
```

Comment:
```
public static final String CLASS_NAME_4_CONNECTION_POSTFIX = ".jdbc.driver.AltibaseConnection";
public static final String CLASS_NAME_4_MESSAGE_CALLBACK_POSTFIX = ".jdbc.driver.AltibaseMessageCallback";
public static final String METHOD_NAME_4_REGISTER_MESSAGE_CALLBACK = "registerMessageCallback";
```

also unused constants

Reply:
They are used at AltibaseOutputReader to handle Altibase class name in the AltibaseDataSource.java. e.g. Altibase.jdbc.driver.AltibaseMessageCallback, Altibase7_1.jdbc.driver.AltibaseMessageCallback